### PR TITLE
Test that the right execution info was logged.

### PR DESCRIPTION
**Pull request by ehlemur-zz**

### CODE REVIEW

**Comment by ehlemur-zz** [comment]
PTAL

**Comment by ehlemur-zz** [comment]
@pbos @kwiberg

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 .*.swp
+*.pyc
```
Comment (kwiberg) [inline .gitignore]: Make this *.py[co] while you're at it?

**Review by kwiberg — APPROVED** [review]
Did this test find any bugs?

**Comment by ehlemur-zz** [comment]
No, it didn't

**Review by kwiberg — APPROVED** [review]

**Pull request merged**